src/game/objects/Player.js

```diff
@@ -36,9 +37,10 @@ export default class Player extends Phaser.Physics.Arcade.Sprite {
     const name = this.displayName();
     this.name = scene.add.text(x - 2 * name.length - 8, y + 22, `${name}`, {
       font: '10px',
-      fill: '#ffffff',
-      backgroundColor: '#000000',
+      fill: this.team?.color ? '#000000' : '#ffffff',
```

Comment:
since the teams have their names - maybe the team's name should be also displayed somewhere (e.g. under the player's name)?

Reply:
i wouldn't care about it now, let's just try it with colors marked